producer/oam.go

```diff
@@ -39,6 +42,25 @@ type UEContext struct {
 
 type UEContexts []UEContext
 
+func HandleOAMPurgeUEContextRequest(supi, reqUri string, msg interface{}) (interface{}, string, interface{}, interface{}) {
+	amfSelf := context.AMF_Self()
+	if ue, ok := amfSelf.AmfUeFindBySupi(supi); ok {
```

Comment:
can we send an error response (ex:404 not found) if supi doesn't exist in AMF? 
please add log if supi is not exist

Reply:
Changes done.